src/Novu/Interfaces/INotificationTemplatesClient.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Novu.DTO;
+using Refit;
+
+namespace Novu.Interfaces;
+
+public record PreferenceSettings(
+    bool Email,
+    bool Sms,
+    bool Chat,
+    bool Push,
+    [property: JsonProperty("in_app")] bool InApp);
+public record Child(string Field,
+    string Value,
+    string Operator,
+    string On);
+
+public record Filter(
+    bool IsNegated,
+    string Type,
+    string Value,
+    Child[] Children);
+
+public record Variable(string Name);
+public record Trigger(
+    string Type,
+    string Identifier,
+    Variable[] Variables,
+    Variable[] SubscriberVariables);
+
+public record NotificationGroup(
+    [property: JsonProperty("_id")] string Id,
+    [property: JsonProperty("_organizationId")] string OrganizationId,
+    [property: JsonProperty("_environmentId")] string EnvironmentId,
+    [property: JsonProperty("_parentId")] string ParentId,
+    string Name);
+public record Step(
```

Comment:
Please can you indent. Many of the records are close to each other in a way that's making it hard to read